cypress/basic/package.json

```diff
@@ -5,7 +5,7 @@
     "test": "cypress run --headed --browser=chrome"
   },
   "devDependencies": {
-    "@axe-core/watcher": "^3.11.1",
+    "@axe-core/watcher": "^3.12.0",
```

Comment:
Does the watcher update need to happen in the same PR? I'd rather separate the separate chunks of work (the headless=new updates make more sense in terms of being required to enable the CI build)

Reply:
Ya there was a fix in 3.12.0 relating to WDIO v7. I thought I might as well update all while I was here